Remove the second time break on parent function.

panzhongxian opened the pull request:
Fix #9

mostafa commented:
Can you add it as a test case to https://github.com/mostafa/goja/blob/debugger/debugger_test.go?

panzhongxian commented:
@mostafa Test case has been added.

mostafa commented on debugger_test.go:
```diff
@@ -18,21 +18,16 @@ func TestDebuggerBreakpoint(t *testing.T) {
-		t.Fatal(err)
-	} else {
-		t.Log("Set breakpoint on line 5")
+	setBreakpointAndLog := func(line int) {
+		if err := debugger.SetBreakpoint("test.js", line); err != nil {
+			t.Fatal(err)
+		} else {
+			t.Logf("Set breakpoint on line %d", line)
+		}
 	}
+	setBreakpointAndLog(3)
+	setBreakpointAndLog(4)
+	setBreakpointAndLog(5)
```
Nice! 👏

mostafa commented on debugger_test.go:
```diff
@@ -184,6 +177,58 @@ func TestDebuggerContinue(t *testing.T) {
+				t.Logf("hit breakpoint on line %d", debugger.Line())
+			}
+		}
+		/*
+			reason := debugger.Continue()
+			// unreachable?
+			t.Fatalf("wrong activation %s", reason)
+		*/
```
If this is not needed, you might want to remove it.

mostafa reviewed (approved):
I think rebasing from latest master would cause the extra commits to go away. Otherwise LGTM.

panzhongxian commented:
> I think rebasing from latest master would cause the extra commits to go away. Otherwise LGTM.

@mostafa I have rebased it and removed the useless comment.
